docs/docs/getting-started.md

```diff
@@ -37,7 +37,7 @@ Use docker-compose to start a local instance of Plane:
 
 ```bash
 cd plane/sample-config/compose
-docker compose up
+docker compose pull && docker compose up
```

Comment:
nit: let's just do these on separate lines, i.e.

```bash
cd plane/sample-config/compose
docker compose pull
docker compose up
```

That way it's easier to debug because the output is directly associated with the command that caused it.